pom.xml

```diff
@@ -437,10 +421,37 @@
           </reportSet>
         </reportSets>
       </plugin>
-     </plugins>
+    </plugins>
   </reporting>
 
   <profiles>
+    <!-- JaCoCo for test coverage -->
+    <profile>
+      <id>jacoco-report</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>default-prepare-agent</id>
+                <goals>
+                  <goal>prepare-agent</goal>
+                </goals>
+              </execution>
+              <execution>
+                <id>default-report</id>
+                <goals>
+                  <goal>report</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
```

Comment:
The other profiles don't have an empty line between them, we should keep it the same way here

Reply:
@lmarniazman
Empty lines removed.Thnaks